src/main/java/nextstep/courses/infrastructure/JdbcSessionRepository.java

```diff
@@ -0,0 +1,101 @@
+package nextstep.courses.infrastructure;
+
+import nextstep.courses.domain.Course;
+import nextstep.courses.domain.Image;
+import nextstep.courses.domain.Image.Dimension;
+import nextstep.courses.domain.ImageType;
+import nextstep.courses.domain.Session;
+import nextstep.courses.domain.Session.SessionDuration;
+import nextstep.courses.domain.SessionPayType;
+import nextstep.courses.domain.SessionRepository;
+import nextstep.courses.domain.SessionState;
+import nextstep.courses.domain.SessionStudent;
+import org.springframework.jdbc.core.JdbcOperations;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+
+import java.sql.PreparedStatement;
+import java.sql.Timestamp;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+@Repository("sessionRepository")
+public class JdbcSessionRepository implements SessionRepository {
+
+    private JdbcOperations jdbcTemplate;
+
+    public JdbcSessionRepository(JdbcOperations jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    @Override
+    public Session save(Session session) {
+        Image image = session.getImage();
+        String imageSql = "insert into image (size, type, width, height, created_at) values(?, ?, ?, ?, ?)";
+        KeyHolder imageKeyHolder = new GeneratedKeyHolder();
+        jdbcTemplate.update(connection -> {
+            PreparedStatement ps = connection.prepareStatement(imageSql, new String[]{"id"});
+            ps.setInt(1, image.getSize());
+            ps.setString(2, image.getType());
+            ps.setInt(3, image.getWidth());
+            ps.setInt(4, image.getHeight());
+            ps.setTimestamp(5, Timestamp.valueOf(LocalDateTime.now()));
+            return ps;
+        }, imageKeyHolder);
```

Comment:
```suggestion
        long imageid = imageSave(session);
```

조금 복잡하네요~ 메소드 분리해도 좋을것 같아요~